rse-index.Rmd

```diff
@@ -13,49 +13,163 @@ github-repo: merely-useful/merely-useful.github.io
+However,
+sustainability isn't just a property of the software:
+it also depends on the skills and culture of its users.
+If a software package is being maintained by a couple of post-docs
+who are being paid a fraction of what they could earn in industry
+and have no realistic hope of promotion because their field doesn't value tool building,
+it doesn't matter whether the package is well tested and easy to install:
+sooner or later (probably sooner)
+it will become [abandonware][abandonware].
+
+### Why isn't all of this already normal?
 
 Nobody argues that research should be irreproducible or unsustainable,
 but "not against it" and actively supporting it are very different things.
 Academia doesn't yet know how to reward people for writing useful software,
 so while you may be thanked,
-the extra effort you put in may not translate into job security or decent pay.
+the effort you put in may not translate into job security or decent pay.
 
-And some people still argue against openness,
-Being open is a big step toward a (non-academic) career path,
-which is where approximately 80% of PhDs go,
-and for those staying in academia,
-open work is cited more often than closed (FIXME: citation).
-However,
-some people still worry that if they make their data and code generally available,
+And some people still worry that if they make their data and code generally available,
 someone else will use it and publish a result they have come up with themselves.
 This is almost unheard of in practice,
-but that doesn't stop people using it as a boogeyman.
-
+but that doesn't stop people using it as a scare tactic.
 Other people are afraid of looking foolish or incompetent by sharing code that might contain bugs.
-This isn't just [impostor syndrome](glossary.html#impostor-syndrome):
-members of marginalized groups are frequently judged more harshly than others (FIXME: CITE).
+This isn't just [impostor syndrome][impostor-syndrone]:
+members of marginalized groups are frequently judged more harshly than others,
+so being wrong in public is much riskier for them.
+
+### Syllabus
 
-## Acknowledgments {#overview-ack}
+This book covers the tools and practices needed to create work that is open, reproducible, and sustainable.
+You can use this book for self-study
+or as the foundation of a one-semester course for graduate students or senior undergraduates.
+By its end,
+you will be able to:
+
+-   Use [branches][branch] to coordinate development of complex software.
+-   Enable users to configure software without modifying it.
+-   Re-run analyses and rebuild packages reproducibly with a single command.
+-   Test your software and know which parts have not yet been tested.
+-   Keep track of what still needs to be done and who's working on it.
+-   Work on remote computers such as computing clusters or the cloud.
+-   Document software so that other people can use it, maintain it, and extend it.
```

Comment:
Maybe this is mostly a personal quirk, but I am not a fan of the expression "other people". Some possible alternatives, just for consideration

"Document software to facilitate its use, maintenance, and extendability."
or
"Document software so it can be easily used, maintained, and extended."